stages/org.osbuild.wpa_supplicant

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+"""
+Write wpa_supplicant configuration file.
```

Comment:
I thought that the general RHEL focus was on NetworkManager?

Reply:
It is. I was fiddling with another distro when I wrote this stage :)